packages/ui-components/src/components/text-field/text-field.tsx

```diff
@@ -162,13 +169,17 @@ export class KvTextField implements ITextField, ITextFieldEvents {
 	}
 
 	private getType(): string {
-		return this.type === EInputFieldType.Number ? EInputFieldType.Text : this.type;
+		return this.isInputMaskEnabled() ? EInputFieldType.Text : this.type;
 	}
 
 	private getTooltipConfig = (): Partial<ITooltip> => {
 		return merge({}, DEFAULT_TEXT_TOOLTIP_CONFIG, this.tooltipConfig ?? {});
 	};
 
+	private isInputMaskEnabled = (): boolean => {
```

Comment:
Would it make sense to change this to a getter?

Reply:
Seems good to me 👍